requirements/optional.txt

```diff
@@ -0,0 +1,16 @@
+# some of these dependencies live on a private git server. If you have the address,
+# configure git with:
+# git config --global url.https://<private hostname>/.insteadOf https://private.repo/
```

Comment:
This is clever... but I'm still not convinced it's a better solution than just defining our private requirements in the same places as the rest of our private setup details.